layouts/partials/head.html

```diff
@@ -11,6 +11,21 @@
 </script>
 <meta charset="utf-8">
 <title>{{ if .Title }}{{ .Title }} - {{ end }}{{ site.Title }}</title>
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "Organization",
+    "url": "https://kubernetes.io",
+    "logo": "https://kubernetes.io/images/favicon.png",
+{{- if not .Site.Params.deprecated }}
+    "potentialAction": {
+      "@type": "SearchAction",
+      "target": {{ printf "%s%s" ("/docs/search/" | absURL) "?q={search_term_string}" }},
```

Comment:
Hi @remyleone @sftim 
Can you describe to me why the docs site needs the "potentialAction" key?
Where is "search_term_string" provided?
Thanks!

Reply:
See https://github.com/kubernetes/website/pull/19344#issue-380905117 @kbhawkey 

This change will let search pages that interpret Google-style linked data add a “search Kubernetes” form onto the results page.

Reply:
Here's an example that (for me) includes a search for `thesaurus.com` on the results page: https://www.google.com/search?q=thesaurus

Reply:
@remyleone it might help to expand the PR description a bit to explain, if you like, the _business impact_ of this PR.

Reply:
Yes, I understand (generally). The json-ld script was added previously. Now you want to add the sitelinks search box,

```
A sitelinks search box is a quick way for people to search your site or app immediately on the search results page. The search box implements real-time suggestions and other features.
```

However you are excluding the deprecated versions of the docs. What are the pitfalls and what is the advantage to the site vs the advantage to another party (commercial)?

Reply:
I'm not sure to understand your question @kbhawkey I'm not sure there is any pitfall except putting the current version as the default search one except for the old one. That being said, people can always switch version on the page.

Reply:
Ah - the previous versions' search pages actually show results from the latest docs. Eg https://v1-17.docs.kubernetes.io/docs/search/?q=configmap

It therefore makes sense only to offer a search for the current minor release.

Reply:
@remyleone , I am asking if there is a reason *not* to add this search (especially because @zacharysarah flagged this as a possible issue). Or, are there strong reasons to add this search?

Reply:
The reason to add it is to have a better UX in google results. I am not an SEO expert, therefore, I don't have any take on the SEO points raised by @zacharysarah

Reply:
OK. Apparently, we can control the addition of this search box from the docs site with:
`<meta name="google" content="nositelinkssearchbox" />`